setuptools/command/easy_install.py

```diff
@@ -2038,14 +2038,19 @@ def chmod(path, mode):
         log.debug("chmod failed: %s", e)
 
 
+class SplitArgs(TypedDict, total=False):
```

Comment:
Should we prefer introducing private terms?

Reply:
Is it intended for devs to assign to `split_args` at any point (and I mean re-assigning to the attribute directly, not mutating the dictionary)? If not, it could be made private. (the same reasoning can be applied to TypeDict used solely in args vs return types)

Reply:
Ideally users should not be importing more symbols in general 😅, and more so from deprecated modules.

Hopefully devs will not have to assign new values, but if they do I think they should simply use a literal dict with the correct keys. Mypy should be able to cope with that right?

Reply:
Yes using a dict literal to assign to a typeddict should be fine. The module being deprecated is even more reason. I'll make them both private.
If a user really needs to they can still import the private symbol for reference.